bc_obps/service/operation_service_v2.py

```diff
@@ -74,7 +100,8 @@ def list_current_users_unregistered_operations(
     @classmethod
     @transaction.atomic()
     def update_status(cls, user_guid: UUID, operation_id: UUID, status: Operation.Statuses) -> Operation:
-        operation = OperationService.get_if_authorized(user_guid, operation_id)
+        # brianna don't use only_fields here--this function is often used in other functions that need more than the only_fields and then that triggers a query for every individual field
```

Comment:
Note to self so I don't try this when doing revisions, will delete comment before merging